pkg/ssh/ssh.go

```diff
@@ -47,7 +48,10 @@ func getSSHCommand(provider *SSHProvider) []string {
 	}
 
 	if provider.Config.ExtraFlags != "" {
-		result = append(result, strings.Split(provider.Config.ExtraFlags, " ")...)
+		flags, err := shellquote.Split(provider.Config.ExtraFlags)
```

Comment:
do we want to ignore the error completely?

Reply:
Right! Fixed it :+1: